docs/preferences.md

```diff
@@ -0,0 +1,147 @@
+# Preferences
+
+## General
+
+The **General** tab, provides general information about the project's status as well as links to discuss the project, report issues, or to learn more about the project.
+
+### Check for updates automatically
+
+When an update is available, users will be provided a notification and the release notes for the upgrade target. This happens whether automatic updates are enabled or not. If this option is enabled, the update will be downloaded and then installed the next time Rancher Desktop is started.
+
+### Allow collection of anonymous statistics to help us improve Rancher Desktop
+
+This option allows Rancher Desktop to collect various usage metrics.
```

Comment:
… We do not collect information about what workloads you run, only how you interact with the application itself.

(Actually, the checkbox doesn't actually do anything since we don't collect anything yet, but we plan to.)